### DISCUSSION TITLE
Pixelr Tracker

### DISCUSSION THREAD

**Discussion by dstachio** [ESPresense/ESPresense#118] [Q&A]
First,

Love the project.  Easy to set up and working great with most of my devices.

I originally bought some Pixelr Trackers a while ago for similar reasons to track.  Unfortunately, they are sending out the same ID but with different MAC addresses.  I got 3 of them currently transmitting all showing md:0000:8 for id but different MAC.  Any way around this possibly?

**Comment by DTTerastar** [comment]
Are they like this: https://www.amazon.com/TrackR-pixel-Bluetooth-Tracking-Compatible/dp/B073HLQBNH?th=1

**Comment by dstachio** [comment]
Yep. Those are them.
